src/libraries/libraries-packages.proj

```diff
@@ -44,6 +44,9 @@
       -->
       <PkgProjects Include="$(PkgDir)*\*.pkgproj" Exclude="$(MSBuildThisFileDirectory)pkg\*Private*\*.pkgproj" />
       <PkgProjects Include="*\pkg\**\*.pkgproj" />
+      <PkgProjects Remove="Microsoft.Extensions.HostFactoryResolver\pkg\Microsoft.Extensions.HostFactoryResolver.Sources.pkgproj" />
```

Comment:
Why bother doing all the other changes if these packages are note going to be built at all in servicing branch?

Reply:
https://github.com/dotnet/runtime/pull/42872/files/0b646c327798f236ce6664f74cecfd4208d39898#diff-db6238b87f150b0d817e43057d645249R45 
is excluding all the private packages from stable, without excluding these they end up being stable non-shipping packages.

Reply:
Yeah, that is what I mean, since we're not building these packages at all on servicing branch, why bother doing all the other changes in their `Directory.Build.props`, etc?

Reply:
But I just saw that you reverted those changes.